cloud/pkg/cloudhub/servers/httpserver/node/nodename.go

```diff
@@ -0,0 +1,43 @@
+package node
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/emicklei/go-restful"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog/v2"
+
+	"github.com/kubeedge/kubeedge/cloud/pkg/common/client"
+)
+
+func HasNodeName(request *restful.Request, response *restful.Response) {
+	// get nodeName
+	nodeName := request.PathParameter("nodename")
+	if nodeName == "" {
+		err := response.WriteErrorString(http.StatusBadRequest, "nodename parameter is required")
+		if err != nil {
+			klog.Errorf("failed to send the task resp to edge , err: %v", err)
+		}
+		return
+	}
+
+	_, err := client.GetKubeClient().CoreV1().Nodes().Get(context.TODO(), nodeName, metav1.GetOptions{})
+	if err != nil {
```

Comment:
if err != nil and !errors.IsNotFound(err) , what should we do here? It seems now we response `Node founded` also for this err?

Reply:
Thank you for pointing out the problem. I have modified it.